catppuccin.tmux

```diff
@@ -89,14 +85,11 @@ main() {
   local window_status_separator=$(get_tmux_option "@catppuccin_window_separator" "")
   setw window-status-separator "$window_status_separator"
 
-  setw window-status-activity-style "fg=${thm_fg},bg=${thm_bg},none"
-  setw window-status-style "fg=${thm_fg},bg=${thm_bg},none"
-
```

Comment:
Just making sure this was not removed by accident.

Reply:
This is needed because in tmux once you set the bg to a solid color, it can't be set to transparent afterwards.

It is better to have the builder decide on the colors and the set them in tmux only once.